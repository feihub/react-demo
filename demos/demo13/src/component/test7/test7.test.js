import { act as domAct } from "react-dom/test-utils";
import { act as testAct, create } from "react-test-renderer";
import Test7 from './test7'

let root;

it('test7', () => {

    domAct(() => {
        testAct(() => {
            root = create(<Test7 />);
        });
    });
    expect(root).toMatchSnapshot();

});

