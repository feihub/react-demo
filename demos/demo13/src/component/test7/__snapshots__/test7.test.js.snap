// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  你好！7
</div>
`;

exports[`test7 1`] = `
<div>
  你好！7
</div>
`;
